web/source/kmwembedded.js

```diff
@@ -18,17 +17,16 @@
   // Flag to control refreshing of a keyboard that is already loaded
   keymanweb.mustReloadKeyboard = true;
   
-  // Skip full page initialization (code not needed for embedded applications)
-  keymanweb.fullInitialization = false;
+  // Skip full page initialization - skips native-mode only code
+  keymanweb.isEmbedded = true;
```

Comment:
Thanks.  I felt like I'd done something like it before but was completely blind to it here for some reason.